Yubico.YubiKey/src/Yubico/YubiKey/SmartCardDeviceInfoFactory.cs

```diff
@@ -67,67 +68,76 @@ public static YubiKeyDeviceInfo GetDeviceInfo(ISmartCardDevice device)
                 ykDeviceInfo.FirmwareVersion = firmwareVersion;
             }
 
-            if (ykDeviceInfo.FirmwareVersion < FirmwareVersion.V4_0_0 && ykDeviceInfo.AvailableUsbCapabilities == YubiKeyCapabilities.None)
+            if (ykDeviceInfo.FirmwareVersion < FirmwareVersion.V4_0_0 &&
+                ykDeviceInfo.AvailableUsbCapabilities == YubiKeyCapabilities.None)
             {
-                ykDeviceInfo.AvailableUsbCapabilities = YubiKeyCapabilities.Oath | YubiKeyCapabilities.OpenPgp | YubiKeyCapabilities.Piv | YubiKeyCapabilities.Ccid;
+                ykDeviceInfo.AvailableUsbCapabilities = YubiKeyCapabilities.Oath | YubiKeyCapabilities.OpenPgp |
+                    YubiKeyCapabilities.Piv | YubiKeyCapabilities.Ccid;
```

Comment:
```suggestion
                ykDeviceInfo.AvailableUsbCapabilities =
                    YubiKeyCapabilities.Oath |
                    YubiKeyCapabilities.OpenPgp |
                    YubiKeyCapabilities.Piv |
                    YubiKeyCapabilities.Ccid;
```

style nit: Typically we favor wrapping each item on its own line if we are going to wrap.